Cargo.toml

```diff
@@ -130,12 +144,12 @@ paste = "1.0"
 
 # For the DB & values layer.
 crossbeam-queue = "0.3"
-hi_sparse_bitset = "0.6" # For buffer pool allocator in the DB
-im = "15.1"              # Immutable data structures
+hi_sparse_bitset = "0.6"                                                    # For buffer pool allocator in the DB
```

Comment:
These format changes seem.. wrong.?

Reply:
Gah, the auto-formatter strikes again. I'll take care of it.

Reply:
Do you want me to just land this and reformat it back?

Reply:
Cleaning up one's own mess is an important didactic tool, but if you have the cycles to do it, then sure, that'd be neat!

Reply:
Probably what we need is a standardized toml format step, since it seems `cargo fmt` doesn't handle it.